components/constellation/constellation.rs

```diff
@@ -2713,13 +2713,8 @@ where
 
         if let Some(webgl_threads) = self.webgl_threads.as_ref() {
             debug!("Exiting WebGL thread.");
-            match webgl_threads.exit() {
-                Ok(_) => {
-                    let _ = webgl_threads_sender.send(());
-                },
-                Err(e) => {
-                    warn!("Exit WebGL Thread failed ({})", e);
-                },
+            if let Err(e) = webgl_threads.exit(webgl_threads_sender) {
+                warn!("Exit WebGL Thread failed ({})", e);
```

Comment:
Small cleanup suggestion:

```suggestion
                warn!("Exit WebGL Thread failed ({e})");
```